### PR TITLE
Ewl 816 create program detail component

### PR DESCRIPTION
**Pull request by ra-phael**
https://eurosportdigital.atlassian.net/browse/EWL-816

### CODE REVIEW

File: src/elements/ProgramDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  color: ${suitGray};
+  * :nth-child(2) {
+    margin-left: 10px;
+    padding-left: 10px;
+
+    padding: 8px;
+  }
+`;
+const StyledIconWrapper = styled.div`
+  padding: ${props => `${props.iconPadding}px`};
```
Comment (val3ntin) [inline src/elements/ProgramDetails/index.js]: sorry, i don't see a way of using this in a future.
once you develop a mobile-first app it's ok to use StyledComponents.
why we do need to set up a specific font size ?

for instance, then after component's usage you would need to subscribe for screen resize / mediachange and set the different font ?

I think once we have styledComponents here - we should follow the same approach.

However, if you want to have e.g. baseFontSize - then i'd advise using  baseFont and provide other values in e.g. `em`s

File: src/elements/ProgramDetails/index.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import ProgramDetails from '.';
+import ChannelIcon from '../ChannelIcon';
+
+describe.only('PlayIcon', () => {
+  it('should render as expected', () => {
+    const wrapper = shallow(<ProgramDetails textDetail="Text details" />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render the default channel icon', () => {
```
Comment (val3ntin) [inline src/elements/ProgramDetails/index.test.js]: rudimental test - it's already in your snapshot i think.

**Comment by ra-phael** [comment]
Thanks for the feedback, originally I added Channel Icon which itself requires a height in px and reproduced the same approach for css properties. Even though accessing props in css with emotion is easy, the opposite is not... indeed we would need an HOC (or hook) to get the viewport size as props as you said.
So I made some changes to improve responsiveness, matching as much as possible legacy and removed the css-related props.
Let me know what you think @val3ntin

File: src/elements/ProgramDetails/index.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+export const StyledText = styled.div`
+  padding: 10px;
+
+  ${medium(css`
+    padding: 15px;
+  `)};
+`;
+
+const ProgramDetails = ({ callsign, textDetail }) => (
+  <StyledDetails>
```
Comment (arturkin) [inline src/elements/ProgramDetails/index.js]: Maybe it's better to add `className` prop here to have the ability to style this component with emotion

Comment (ra-phael) [reply src/elements/ProgramDetails/index.js]: Added! :)

**Pull request merged**